tests/ConnectionTest.php

```diff
@@ -46,4 +46,16 @@ public function test_fetching_consumers_default()
 
         $this->assertEquals('response', $response);
     }
+
+    public function test_set_configurations_default()
+    {
+        $heartbeat = $this->app['config']->get('queue.connections.rabbitmq.heartbeat');
+        $this->assertEquals(15, $heartbeat);
+
+        $connectionTimeout = $this->app['config']->get('queue.connections.rabbitmq.connection_timeout');
+        $this->assertEquals(30, $connectionTimeout);
+
+        $readWriteTimeout = $this->app['config']->get('queue.connections.rabbitmq.read_write_timeout');
+        $this->assertEquals(30, $readWriteTimeout);
```

Comment:
with this you're testing the functionality of the `Config` class, while what you should be testing is the service provider's initialisation of the `Connection` object, you can achieve this by mocking `AMQPStreamConnection`. You need to know which configuration params it used so there should be two tests:

1. connection created with default params
2. changing config params and verifying whether it creates the connection with the changed params